facilitator/src/manifest.rs

```diff
@@ -200,8 +200,7 @@ pub struct DataShareProcessorGlobalManifestV1 {
 pub enum DataShareProcessorSpecificManifest {
```

Comment:
The only thing this enum was for was dispatching method calls to either `struct DataShareProcessorSpecificManifestV1` or `struct DataShareProcessorSpecficManifestV2`. Now that there's only one struct, I think you can rename it to `struct DataShareProcessorSpecificManifest`, move the methods from the enum to the struct (eliminating the now-pointless `match` blocks) and delete the enum.

Reply:
Sure -- I left the enum originally to not wipe out the versioning, but YAGNI very much applies here. Done.